mne_bids/write.py

```diff
@@ -309,8 +320,17 @@ def _scans_tsv(raw, raw_fname, fname, overwrite=False, verbose=True):
     meas_date = raw.info['meas_date']
     if isinstance(meas_date, (tuple, list, np.ndarray)):
         meas_date = meas_date[0]
-        acq_time = datetime.fromtimestamp(
-            meas_date).strftime('%Y-%m-%dT%H:%M:%S')
+        # This is equivalent to
+        # acq_time = datetime.fromtimestamp(
+        #     meas_date).strftime('%Y-%m-%dT%H:%M:%S')
+        # This is a windows datetime bug for timestamp < 0. The negative value
```

Comment:
didn't you say above that this number was 86400? why is it different here?

Reply:
I don't know who tried 86400 and put that in the comment originally but it's wrong, only negative values cause an error. Maybe it changed.